gulp/tasks/lint.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var $ = require( 'gulp-load-plugins' )();
+var config = require( '../config' ).lint;
```

Comment:
Should be `var lint`, see https://nodejs.org/api/child_process.html for example of naming style.